granary/test/testdata/comment.mf2.json

```diff
@@ -1,5 +1,5 @@
 {
-  "type": ["h-entry", "p-comment"],
+  "type": ["h-entry", "h-as-comment"],
```

Comment:
ok, i get the type vs property thing...so if we're losing `p-comment` here, should it show up in `properties` instead?